src/app/containers/Manifest/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
+import Helmet from 'react-helmet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+const ManifestContainer = () => {
+  const { manifestPath, service } = useContext(ServiceContext);
+  return (
+    <Helmet>
+      {manifestPath ? (
```

Comment:
👋 Hey, I think we can require the `manifestPath` to render, maybe do something like

```
const ManifestContainer = () => {
  const { manifestPath, service } = useContext(ServiceContext);
	
  if (!manifestPath) {
	return null;
  }

  return (
    <Helmet>
      <link rel="manifest" href={`/${service}${manifestPath}`} />
    </Helmet>
  );
};
```

Reply:
Makes sense.